android/modules/geolocation/src/java/ti/modules/titanium/geolocation/TiLocation.java

```diff
@@ -328,7 +328,9 @@ private KrollDict buildAddress(JSONObject place)
 		address.put(TiC.PROPERTY_CITY, place.optString(TiC.PROPERTY_CITY, ""));
 		address.put(TiC.PROPERTY_REGION1, ""); // AdminArea
 		address.put(TiC.PROPERTY_REGION2, ""); // SubAdminArea
-		address.put(TiC.PROPERTY_POSTAL_CODE, place.optString("zipcode", ""));
+		address.put(TiC.PROPERTY_POSTAL_CODE, place.optString(TiC.PROPERTY_ZIP_CODE, ""));
+		address.put(TiC.PROPERTY_ZIP_CODE, place.optString(TiC.PROPERTY_ZIP_CODE, ""));
```

Comment:
so it looks like zipcode is more "common", but taking a step back, which should be deprecated: `zipcode` or `postalCode`? Thinking in re: to i18n, `postalCode` is probably less US-centric.

Reply:
Thats a bit late now :( Will try but may needs to go 8.0.0 then. Bit late here